platform-includes/performance/enable-automatic-instrumentation/javascript.mdx

```diff
@@ -19,3 +19,5 @@ Sentry.init({
   tracesSampleRate: 1.0,
 });
 ```
+
+<PlatformContent includePath="performance/automatic-instrumentation-custom-routing" />
```

Comment:
l: should this documentation live under "Automatic Instrumentation"? Isn't this rather fitting for "Custom Instrumentation"? 

I guess there's a case for both, given we'Re mentioning `browserTracingIntegration` directly above but maybe it's enough to link to custom in the "Autmatic Instrumentation" page?

Reply:
yeah, IMHO I'd generally restructure this a bit in a follow up - we should rethink this with a bit of time! Currently that's basically the "browser tracing" page, in reality, even though the name is not necessarily clear on that.